source/Calamari.Tests/Fixtures/Nginx/NginxFixture.cs

```diff
@@ -285,5 +294,33 @@ public void TestLocationsUnsuitableForFilenames()
             // Ensure the filename matches the expected format.
             Assert.IsTrue(Directory.GetFiles(Path.Combine(tempDirectory, "conf"))[0].EndsWith($"{nginxServer.VirtualServerName}.conf"));
         }
+
+        (string, string, string) GetCertificateDetails()
+        {
+#if NETCORE
+            var chainCertFilePath = TestEnvironment.GetTestPath("Helpers", "Certificates", "SampleCertificateFiles", "3-cert-chain.pfx");
```

Comment:
`net4x` and `mono` does not support  `X509Certificate2Collection` and other stuff done here to get data from our test cert.